@here/olp-sdk-authentication/oauth-requester

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env node
```

Comment:
Could you please move that line to the bottom?

Reply:
Unfortunately no, because it's mandated by the shell to be the very first line. See also the docs for adding binaries to files.